include/taskolib/SequenceManager.h

```diff
@@ -105,7 +108,7 @@ class SequenceManager
      * \exception Error is thrown if the sequence folder cannot be created.
      */
     Sequence create_sequence(gul14::string_view label = "",
-        SequenceName name = SequenceName{}) const;
+        SequenceName name = SequenceName{});
```

Comment:
At first I removed const qualifier from list_sequences() because of the store_sequence call in the end of the function. Then I decided to ignore this specific case and differentiate between store_sequence and store_sequence_impl. Then I forgot to give the function the const qualifier back. I will do so now